Octokit.Tests.Integration/Clients/TeamsClientTests.cs

```diff
@@ -445,10 +446,14 @@ public async Task UpdatesTeam()
             {
                 var teamName = Helper.MakeNameWithTimestamp("updated-team");
                 var teamDescription = Helper.MakeNameWithTimestamp("updated description");
+
+                // setting TeamPermission.Admin fails with Octokit.ApiValidationException : Setting team permission to admin is no longer supported
+                var teamPermission = TeamPermission.Push;
```

Comment:
It seems to me like admin is not being accepted as permission anymore, even though the docs say it can be.

I can get this test passing with Pull and Push values, but passing admin just fails with 

setting TeamPermission.Admin fails with Octokit.ApiValidationException : Setting team permission to admin is no longer supported